### PR TITLE
Add Phase I heavy-ion relval workflows

### PR DESCRIPTION
**Pull request by mandrenguyen**
Adding MC workflows to test --era Run2_2017 with --scenario Heavyions.
Adding wfs are 150, 150.1, 150.2 and 150.3
The latter 3 are migrated from Run 1 wfs 140.1, 140.2 and 140.4
All Hydjet MinBias wfs have been changed to Hydjet impact parameter = 12 fm, which will take care of the memory limitations in running heavy-ion events in relval.
The event overlay wfs (300-302) have not been migrated yet.  This will wait for when b=12 GEN-SIM has been produced in a pre-release relval.  

The validation and DQM sequences also had to be brought into sync with their pp Phase I equivalents. 
To get these sequences to run I modified the heavy-ion HCAL local reco such that it now runs both hbheprereco and hbhereco, as is the case in pp.
I'm not really sure why this was ever different in the first place though. 
Feedback on this would be appreciated.

### CODE REVIEW

**Comment by kpedro88** [comment]
Uh, changes in HCAL reco for heavy ions should probably be considered carefully. `hbhereco` is actually the `HBHEIsolatedNoiseReflagger`, and I don't know if this has any relevance or desirability for HI. I doubt it is the right approach if DQM/validation is the only issue.

@igv4321 @hatakeyamak @deguio please take a look

**Comment by mandrenguyen** [comment]
@kpedro88 Be that as it may, it would be good to know whether this was actually a conscious decision to exclude the isolated rechit flagger for the heavy-ion reco.  This was not something I recall implementing, at least intentionally.  It's not obvious to me why such a piece of code should be susceptible to failure at high multiplicity, even if it should be less effective.

**Comment by kpedro88** [comment]
@mandrenguyen needs rebase

also, can you update the PR title to be more specific? (i.e. workflows for heavy ions)

**Comment by perrotta** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -628,25 +628,30 @@ def genS(fragment,howMuch):
...
+#hiDefaults2018=merge([hiAlca2018,{'--scenario':'HeavyIons','-n':2,'--beamspot':'Realistic50ns13TeVCollision','--geometry':'DB:extended'}])
+hiDefaults2011=merge([hiAlca2011,{'--scenario':'HeavyIons','-n':2}])
+hiDefaults2015=merge([hiAlca2015,{'--scenario':'HeavyIons','-n':2}])
+hiDefaults2018=merge([hiAlca2018,{'--scenario':'HeavyIons','-n':2,'--geometry':'DB:Extended'}])
 
-steps['HydjetQ_MinBias_5020GeV']=merge([{'-n':1},hiDefaults,genS('Hydjet_Quenched_MinBias_5020GeV_cfi',U2000by1)])
-steps['HydjetQ_MinBias_5020GeVINPUT']={'INPUT':InputInfo(dataSet='/RelValHydjetQ_MinBias_5020GeV/%s/GEN-SIM'%(baseDataSetRelease[9],),location='STD',split=5)}
 
-steps['HydjetQ_MinBias_2760GeV']=merge([{'-n':1},hiDefaults2011,genS('Hydjet_Quenched_MinBias_2760GeV_cfi',U2000by1)])
+steps['HydjetQ_B12_5020GeV_2011']=merge([{'-n':1,'--beamspot':'RealisticHI2011Collision'},hiDefaults2011,genS('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
```
Comment (perrotta) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: The generator fragment does not exist in  Configuration.Generator
Either add, or use an existing one

Comment (mandrenguyen) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: @perrotta :  I went to go add the missing fragment, but it looks like I added a whole bunch of unrelated commits.  Any advide how to fix this?  Or should I just start a new PR from scratch and close this one?

Comment (perrotta) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: Just rebase this one
(possible tutorial in http://cms-sw.github.io/tutorial-resolve-conflicts.html)
